### PR TITLE
MINOR: Added second path for add object example to show cube behind b…

### PR DESCRIPTION
**Pull request by ninok**
…uildings

### CODE REVIEW

File: @here/harp-examples/src/threejs_add-object.ts [hunk 1]
```diff
@@ -40,14 +40,34 @@ export namespace ThreejsAddSimpleObject {
...
+    const prePassMaterial = new THREE.MeshStandardMaterial({
+        color: 0x00ff00fe,
+        opacity: 0.5,
+        depthTest: false,
+        transparent: true
+    });
     const material = new THREE.MeshStandardMaterial({
-        color: 0x00ff00fe
+        color: 0x00ff00fe,
+        opacity: 1.9,
```
Comment (nzjony) [inline @here/harp-examples/src/threejs_add-object.ts]: 1.9 opacity?

File: @here/harp-examples/src/threejs_add-object.ts [hunk 2]
```diff
@@ -40,14 +40,34 @@ export namespace ThreejsAddSimpleObject {
...
+        const cube = new THREE.Object3D();
+
+        const prePassMesh = new THREE.Mesh(geometry, prePassMaterial);
+        prePassMesh.renderOrder = Number.MAX_SAFE_INTEGER - 1;
+        cube.add(prePassMesh);
+
+        const mesh = prePassMesh.clone();
         mesh.renderOrder = Number.MAX_SAFE_INTEGER;
-        return mesh;
+        mesh.material = material;
```
Comment (nzjony) [inline @here/harp-examples/src/threejs_add-object.ts]: does clone not already do this?

File: @here/harp-examples/src/threejs_add-object.ts (hunk 1 above)
Comment (ninok) [reply @here/harp-examples/src/threejs_add-object.ts]: ups, should be 0.9

File: @here/harp-examples/src/threejs_add-object.ts (hunk 2 above)
Comment (ninok) [reply @here/harp-examples/src/threejs_add-object.ts]: no, geometry and material are reused when cloning a mesh.

Comment (nzjony) [reply @here/harp-examples/src/threejs_add-object.ts]: yes of course, you are changing the material....😅

Comment (nzjony) [reply @here/harp-examples/src/threejs_add-object.ts]: in saying that, you could avoid 2 lines and just use new THREE.MESH(geometry, material), would save a line

Comment (ninok) [reply @here/harp-examples/src/threejs_add-object.ts]: ah, true. Will do

**Pull request merged**